public/help/locale-preferences.html

```diff
@@ -0,0 +1,8 @@
+<h4>Locale Preferences</h4>
+
+<dl id="help">
+  <dt>Set prefered locale</dt>
+  <dd>
+This preference allows you to select your prefered language for email messages that the Archive sends to you. The templates for these messages are currently being updated and translated by our volunteers. This is a work in progress; not all messages will be available in languages other than English at this time. If the template for that email has not yet been translated to your language, it will be sent in English.
```

Comment:
Should be "preferred." Also, a couple little phrasing tweaks: "This option allows you to select your preferred language for email messages the Archive sends you." and "If the template for an email has not yet been translated"